cmd/operator-sdk/build/cmd.go

```diff
@@ -232,17 +252,15 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 			}
 		}
 
-		log.Infof("Building test Docker image %s", image)
+		log.Infof("Building test OCI image %s", image)
 
-		testDbArgs := []string{"build", ".", "-f", testDockerfile, "-t", image, "--build-arg", "NAMESPACEDMAN=" + namespacedManBuild, "--build-arg", "BASEIMAGE=" + baseImageName}
-
-		if dockerBuildArgs != "" {
-			splitArgs := strings.Fields(dockerBuildArgs)
-			testDbArgs = append(testDbArgs, splitArgs...)
+		testImageBuildArgs := fmt.Sprintf("%s --build-arg NAMESPACEDMAN=%s --build-arg BASEIMAGE=%s", imageBuildArgs, namespacedManBuild, baseImageName)
```

Comment:
```suggestion
		testImageBuildArgs := fmt.Sprintf("--build-arg NAMESPACEDMAN=%s --build-arg BASEIMAGE=%s", namespacedManBuild, baseImageName)
```
After changing `createBuildCommand`'s signature.